src/chrome/content/rules/Spiegel.xml

```diff
@@ -110,27 +62,12 @@ Fetch error: http://ophirum.spiegel.de/ => https://ophirum.spiegel.de/: (51, "SS
 			<test url="http://www.spiegel.de/pics/43/0,1020,308043,00.jpg" />
 			<test url="http://www.spiegel.de/static/sys/dimensionspixel.gif" />
 
+	<securecookie host=".+" name=".+" />
 
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.spiegel\.de$" name="^_540p$" /-->
-	<!--securecookie host="^abo\.spiegel\.de$" name="^aboshop-common$" /-->
-	<!--securecookie host="^count\.spiegel\.de$" name="^NETMIND_SID$" /-->
-	<!--securecookie host="^magazin\.spiegel\.de$" name="^BIGipServer[\w~-]+$" /-->
-	<!--securecookie host="^ophirum\.spiegel\.de$" name="^PHPSESSID$" /-->
-
-	<securecookie host="^(?:abo|count|magazin|ophirum)\.spiegel\.de$" name=".+" />
-
-
-	<!--	- Cert only matches www
-		- Unsupported paths redirect to http
-		- cdn\d?: Akamai
-				-->
-	<rule from="^http://cdn\d?\.spiegel\.de/"
-		to="https://www.spiegel.de/" />
```

Comment:
Why did you remove this?

Reply:
Because the first step when i am cleaning up old rulesets is deleting all special rewrites to see if it works without them. Also there are no test urls to be found , neither in google nor on the spiegel website.

Reply:
I can find many examples when browsing on the site. With the rule they get secured, but without it you can see the requests to `cdn([1-4])?`.